### ISSUE TITLE
Re-using the Lively browser as a general server-side file browser

### ISSUE THREAD

**Issue by rickmcgeer** [LivelyKernel/lively.next#1507]
**What would you like to achieve?**
This issue one of the cases of https://github.com/LivelyKernel/lively.next/issues/1487.  Briefly, what I'd like to do is incorporate a server-side file browser in Galyleo.  It occurred to me that we already have a terrific server-side file browser: the Lively module browser.  All we need to do is:

1. Change the root of the browser from the lively root directory to something else;
2. Set the action on a double click on a leaf file (and/or turn the editor into a JSON editor)

All of this seems doable, and I'm prepared to do it, I just need pointers to the code and perhaps some commentary as I go (the usual way I have of finding and playing with Lively code doesn't work for elements of the Lively UI! :-)

**Version**: Please paste the lively.next commit on which the problem occurred here (use the copy button of the Version Checker in the bottom left corner).
97874fc6b9ca2be74bda64de4c9a687d3e8d4f8f

**Comment by merryman** [comment]
Did you attempt the following? If so, maybe its easier to enhance this file browser?

![file browser](https://github.com/LivelyKernel/lively.next/assets/1296388/8a1a2d26-3630-4067-9e4d-2effc95f938c)

**Comment by rickmcgeer** [comment]
Yes, that's exactly it, and I found the code!
Let me tell you what's motivating this.  It turns out that the JupyterLab extension is becoming more and more problematic to deal with.  The central issue is that the API is very complex and quite dynamic, and the surface that the extension must meet is large and dynamic. 
Worse, we can't compile environments until we make the extension compatible.
However, we've been able to plug RStudio and VSCode in easily, and of course those applications don't have any idea that they're instantiated by JupyterHub.  So what I want to do is bring up Galyleo as a standalone server.   The only change we need to make to Galyleo is to implement a file browser over the filesystem to handle loads and saves...
